Multithreaded build also doesn't work
```
Traceback (most recent call last):
  File "/home/batman/threepilot/selfdrive/controls/lib/longitudinal_mpc_lib/long_mpc.py", line 10, in <module>
    from openpilot.selfdrive.car.interfaces import ACCEL_MIN
  File "/home/batman/threepilot/openpilot/selfdrive/car/interfaces.py", line 15, in <module>
    from openpilot.selfdrive.controls.lib.events import Events
  File "/home/batman/threepilot/openpilot/selfdrive/controls/lib/events.py", line 7, in <module>
    import cereal.messaging as messaging
  File "/home/batman/threepilot/cereal/messaging/__init__.py", line 2, in <module>
    from .messaging_pyx import Context, Poller, SubSocket, PubSocket, SocketEventHandle, toggle_fake_events, \
ModuleNotFoundError: No module named 'cereal.messaging.messaging_pyx'
```

--- a/SConstruct
+++ b/SConstruct
@@ -364,8 +364,9 @@ else:
   cereal = [File('#cereal/libcereal.a')]
   messaging = [File('#cereal/libmessaging.a')]
   visionipc = [File('#cereal/libvisionipc.a')]
+messaging_python = [File('#cereal/messaging/messaging_pyx.so')]
 
-Export('cereal', 'messaging', 'visionipc')
+Export('cereal', 'messaging', 'messaging_python', 'visionipc')
 
 # Build rednose library and ekf models
 

--- a/selfdrive/controls/lib/lateral_mpc_lib/SConscript
+++ b/selfdrive/controls/lib/lateral_mpc_lib/SConscript
@@ -32,7 +32,10 @@ generated_files = [
   f'{gen}/Makefile',
 
   f'{gen}/main_lat.c',
+  f'{gen}/main_sim_lat.c'm
   f'{gen}/acados_solver_lat.h',
+  f'{gen}/acados_sim_solver_lat.h',
+  f'{gen}/acados_sim_solver_lat.c',
   f'{gen}/acados_solver.pxd',
 
   f'{gen}/lat_model/lat_expl_vde_adj.c',

--- a/selfdrive/controls/lib/lateral_mpc_lib/SConscript
+++ b/selfdrive/controls/lib/lateral_mpc_lib/SConscript
@@ -32,7 +32,7 @@ generated_files = [
   f'{gen}/Makefile',
 
   f'{gen}/main_lat.c',
-  f'{gen}/main_sim_lat.c'm
+  f'{gen}/main_sim_lat.c',
   f'{gen}/acados_solver_lat.h',
   f'{gen}/acados_sim_solver_lat.h',
   f'{gen}/acados_sim_solver_lat.c',

--- a/selfdrive/controls/lib/longitudinal_mpc_lib/SConscript
+++ b/selfdrive/controls/lib/longitudinal_mpc_lib/SConscript
@@ -38,7 +38,10 @@ generated_files = [
   f'{gen}/Makefile',
 
   f'{gen}/main_long.c',
+  f'{gen}/main_sim_long.c',
   f'{gen}/acados_solver_long.h',
+  f'{gen}/acados_sim_solver_long.h',
+  f'{gen}/acados_sim_solver_long.c',
   f'{gen}/acados_solver.pxd',
 
   f'{gen}/long_model/long_expl_vde_adj.c',

--- a/selfdrive/controls/lib/longitudinal_mpc_lib/SConscript
+++ b/selfdrive/controls/lib/longitudinal_mpc_lib/SConscript
@@ -1,4 +1,4 @@
-Import('env', 'envCython', 'arch', 'cereal')
+Import('env', 'envCython', 'arch', 'messaging_python')
 
 gen = "c_generated_code"
 
@@ -66,7 +66,7 @@ lenv.Clean(generated_files, Dir(gen))
 generated_long = lenv.Command(generated_files,
                               source_list,
                               f"cd {Dir('.').abspath} && python3 long_mpc.py")
-lenv.Depends(generated_long, cereal)
+lenv.Depends(generated_long, messaging_python)
 
 lenv["CFLAGS"].append("-DACADOS_WITH_QPOASES")
 lenv["CXXFLAGS"].append("-DACADOS_WITH_QPOASES")

